texmf/tex/latex/latex-lab/documentmetadata-support.ltx

```diff
@@ -0,0 +1,273 @@
+%%
+%% This is file `documentmetadata-support.ltx',
+%% generated with the docstrip utility.
+%%
+%% The original source files were:
+%%
+%% documentmetadata-support.dtx  (with options: `code')
```

Comment:
This is a generated file, does that need to be checked in?

Reply:
My gut feeling is that it is better to use search than to require us to update what is in texmf each time something changes with these files. I think there is a testfiles dir that is for checksearch=true if not we could make one and have the tests that needs this placed in there to keep the earch case at a minimum

Reply:
I've adjusted